code/modules/clothing/masks/gasmask.dm

```diff
@@ -246,16 +246,17 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 	visor_toggling()
 
 // ********************************************************************
-
 //Plague Dr suit can be found in clothing/suits/bio.dm
+
 /obj/item/clothing/mask/gas/plaguedoctor
 	name = "plague doctor mask"
 	desc = "A modernised version of the classic design, this mask will not only filter out toxins but it can also be connected to an air supply."
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS | GAS_FILTERING
```

Comment:
`/obj/item/clothing/mask/gas/plaguedoctor` is a subtype of `/obj/item/clothing/mask/gas`, the parent. Being a subtype of a parent will make it inherit all the behaviors from it, since all of this is already defined on the parent you don't need to add this line

Reply:
That makes sense! Thank you for telling me that. So all subtypes will inherit from their parents? With the hat being a subtype of the hood for example, does that mean I need to make it stand alone in order for it not to inherit the qualities of the biohood? I could move the flags from biohood to the mask, in order to make that resistant to disease, and keep the hat as just a hat.

Reply:
yes you could make it a subtype of clothing head, but I think it's meant to have qualities of the bio hood so you can just disable the vars you don't want while it's a subtype of bio hood.
So you can either 1. Keep it as a subtype of biohood and disable the stuff you don't want, or 2. Make it a subtype of clothing hat/head (whatever it is) and add the behaviors you want.